apps/minifront/src/components/swap/swap-form/token-swap-input.tsx

```diff
@@ -42,7 +51,18 @@ export const TokenSwapInput = () => {
     assetOut,
     setAssetOut,
     balancesResponses,
+    candlestickData,
+    loadCandlestick,
+    latestKnownBlockHeight,
   } = useStoreShallow(tokenSwapInputSelector);
+
+  useEffect(() => {
+    if (!assetIn || !assetOut) return;
+    const ac = new AbortController();
```

Comment:
Are candlestick requests very expensive? Aka, is there any meaningful downside to excluding the abort controller stuff for the sake of simpler code?

Reply:
they can be many thousands of records. right now it's fixed, but eventually the size will be configurable. the abortcontroller was previously in the slice, but moved here as i needed some more detailed control - i could probably move it back into the slice